modules/programs/powerline-go.nix

```diff
@@ -0,0 +1,271 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.powerline-go;
+
+  moduleNames = [
+    "aws"
+    "cwd"
+    "docker"
+    "docker-context"
+    "dotenv"
+    "duration"
+    "exit"
+    "git"
+    "gitlite"
+    "hg"
+    "host"
+    "jobs"
+    "kube"
+    "load"
+    "newline"
+    "nix-shell"
+    "node"
+    "perlbrew"
+    "perms"
+    "plenv"
+    "root"
+    "shell-var"
+    "shenv"
+    "ssh"
+    "svn"
+    "termtitle"
+    "terraform-workspace"
+    "time"
+    "user"
+    "venv"
+    "vgo"
+  ];
+
+  optionSpecs = [
+    {
+      name = "alternateSshIcon";
+      description = "Show the older, original icon for SSH connections";
+      type = types.bool;
+    }
+    {
+      name = "colorizeHostname";
+      description =
+        "Colorize the hostname based on a hash of itself, or use the PLGO_HOSTNAMEFG and/or PLGO_HOSTNAMEBG env vars.";
```

Comment:
Try to keep lines at a maximum of 80 characters. Descriptions are docbook formatted so line breaks won't cause problems.

By the way, all these descriptions seem to come from the powerline-go documentation, which as far as I can tell is licensed under GPL-3 and can unfortunately not be used in this project, which is MIT licensed.

Reply:
fixed